pkg/oc/cli/cmd/tag.go

```diff
@@ -145,6 +145,15 @@ func determineSourceKind(f *clientcmd.Factory, input string) string {
 		return "DockerImage"
 	}
 
+	glog.Warningf("%v", err)
```

Comment:
I think the convention here is to use:
`glog.V(...).Infof("error while determining source kind: %v")` or something along these lines